oscar_odin/mappings/constants.py

```diff
@@ -17,6 +17,7 @@
 PRODUCT_META_TITLE = "Product.meta_title"
 PRODUCT_META_DESCRIPTION = "Product.meta_description"
 PRODUCT_IS_DISCOUNTABLE = "Product.is_discountable"
+PRODUCT_PRIORITY = "Product.priority"
```

Comment:
Do we have priority field for product model?

Reply:
yup